### PR TITLE
Update index.js to add Mantle Chain Pools

### PR DESCRIPTION
**Pull request by VarinderClearpool**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Clearpool Finance

##### Website Link:
clearpool.finance

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/clearpool exports TVL: 
        
 
 
    ethereum-staking          40.51 M
    staking                   40.51 M
    borrowed                  12.41 M
    optimism-borrowed         6.65 M
    ethereum-borrowed         5.30 M
    ethereum                  1.39 M
    optimism                  1.28 M
    mantle-borrowed           270.72 k
    polygon_zkevm-borrowed    182.23 k
    mantle                    64.49 k
    polygon                   41.64 k
    polygon_zkevm             34.98 k
    polygon-borrowed          0
    
    total                    2.81 M

**Comment by VarinderClearpool** [comment]
The above return is correct for the PR changes, thank you!

**Comment by llamatester** [comment]
Error while running adapter at projects/clearpool: 
        
 
  
    
    
    Error: 
        Unknown chain(s): avalanche
        Note: if you think that the chain is correct but missing from our list, please add it to 'projects/helper/chains.json' file

**Comment by llamatester** [comment]
Error while running adapter at projects/clearpool: 
        
 
  
    
    
    Error: 
        Unknown chain(s): avalanche
        Note: if you think that the chain is correct but missing from our list, please add it to 'projects/helper/chains.json' file

File: projects/clearpool/index.js [hunk 1]
```diff
@@ -10,14 +10,16 @@ const singleStakingContracts = ["0x629E39da1Db5654fe59cAE31d48CAEBB8dC2A9c6"];
...
-  ethereum: { factory: '0xde204e5a060ba5d3b63c7a4099712959114c2d48', fromBlock: 14443222 },
-  polygon_zkevm: { factory: '0xCE3Fec90A05992dF1357651FEF6D143FeeC7Ca16', fromBlock: 302970 },
-  polygon: { factory: '0x215CCa938dF02c9814BE2D39A285B941FbdA79bA', fromBlock: 31128013 },
-  optimism: { factory: '0x99C10A7aBd93b2db6d1a2271e69F268a2c356b80', fromBlock: 107128813 },
+  ethereum: { factory: '0xde204e5a060ba5d3b63c7a4099712959114c2d48', fromBlock: 14443222 }, // Permissionless Pools
+  polygon_zkevm: { factory: '0xCE3Fec90A05992dF1357651FEF6D143FeeC7Ca16', fromBlock: 302970 }, // Permissionless Pools
+  polygon: { factory: '0x215CCa938dF02c9814BE2D39A285B941FbdA79bA', fromBlock: 31128013 }, // Permissionless Pools
+  optimism: { factory: '0x99C10A7aBd93b2db6d1a2271e69F268a2c356b80', fromBlock: 107128813 }, // Permissionless Pools
+  mantle: { factory: '0xB217D93a8f6A4b7861bB2C865a8C22105FbCdE41', fromBlock: 49691965 }, // Permissionless Pools
+  avalanche: { factory: '0x8E557363AC9E5cbf09A2616A302CA3c8f6ab2b7A', fromBlock: 43675420 }, // Clearpool Credit Vaults
```
Comment (VarinderClearpool) [inline projects/clearpool/index.js:22]:
```suggestion
  avax: { factory: '0x8E557363AC9E5cbf09A2616A302CA3c8f6ab2b7A', fromBlock: 43675420 }, // Clearpool Credit Vaults
```

**Comment by llamatester** [comment]
The adapter at projects/clearpool exports TVL: 
        
 
 
    ethereum-staking          40.15 M
    staking                   40.15 M
    borrowed                  12.41 M
    optimism-borrowed         6.65 M
    ethereum-borrowed         5.30 M
    ethereum                  1.39 M
    optimism                  1.28 M
    mantle-borrowed           269.24 k
    polygon_zkevm-borrowed    182.04 k
    mantle                    64.14 k
    polygon                   41.64 k
    polygon_zkevm             34.94 k
    polygon-borrowed          0
    avax-borrowed             0
    avax                      0
    
    total                    2.81 M

File: projects/clearpool/index.js [hunk 2]
```diff
@@ -10,14 +10,16 @@ const singleStakingContracts = ["0x629E39da1Db5654fe59cAE31d48CAEBB8dC2A9c6"];
...
-  ethereum: { factory: '0xde204e5a060ba5d3b63c7a4099712959114c2d48', fromBlock: 14443222 },
-  polygon_zkevm: { factory: '0xCE3Fec90A05992dF1357651FEF6D143FeeC7Ca16', fromBlock: 302970 },
-  polygon: { factory: '0x215CCa938dF02c9814BE2D39A285B941FbdA79bA', fromBlock: 31128013 },
-  optimism: { factory: '0x99C10A7aBd93b2db6d1a2271e69F268a2c356b80', fromBlock: 107128813 },
+  ethereum: { factory: '0xde204e5a060ba5d3b63c7a4099712959114c2d48', fromBlock: 14443222 }, // Permissionless Pools
+  polygon_zkevm: { factory: '0xCE3Fec90A05992dF1357651FEF6D143FeeC7Ca16', fromBlock: 302970 }, // Permissionless Pools
+  polygon: { factory: '0x215CCa938dF02c9814BE2D39A285B941FbdA79bA', fromBlock: 31128013 }, // Permissionless Pools
+  optimism: { factory: '0x99C10A7aBd93b2db6d1a2271e69F268a2c356b80', fromBlock: 107128813 }, // Permissionless Pools
+  mantle: { factory: '0xB217D93a8f6A4b7861bB2C865a8C22105FbCdE41', fromBlock: 49691965 }, // Permissionless Pools
+  avax: { factory: '0x8E557363AC9E5cbf09A2616A302CA3c8f6ab2b7A', fromBlock: 43675420 }, // Clearpool Credit Vaults
```
Comment (VarinderClearpool) [inline projects/clearpool/index.js:22]:
```suggestion
```

**Comment by llamatester** [comment]
The adapter at projects/clearpool exports TVL: 
        
 
 
    ethereum-staking          39.98 M
    staking                   39.98 M
    borrowed                  12.40 M
    optimism-borrowed         6.65 M
    ethereum-borrowed         5.30 M
    ethereum                  1.39 M
    optimism                  1.28 M
    mantle-borrowed           269.39 k
    polygon_zkevm-borrowed    182.05 k
    mantle                    64.17 k
    polygon                   41.66 k
    polygon_zkevm             34.94 k
    polygon-borrowed          0
    
    total                    2.81 M

**Comment by g1nt0ki** [comment]
@VarinderClearpool thanks for the PR

**Comment by VarinderClearpool** [comment]
Thank you @g1nt0ki

**Pull request merged**